constants/deco_constants.asm

```diff
@@ -54,16 +54,18 @@ const_value = 1
 	const SET_UP_ORNAMENT
 	const PUT_AWAY_ORNAMENT
 
+__deco_value__ = 0
+
 deco: MACRO
 	const DECO_\1
-	enum DECOFLAG_\1
+DECOFLAG_\1 EQU __deco_value__
```

Comment:
Can't this just become `DECOFLAG_\1 EQU const_value - 1`?

Reply:
No, because amid the `deco` uses are `const` uses: `const BEDS`, `const CARPETS`, `const PLANTS`, etc. So the bed `DECOFLAG`s are 2 less than the bed `DECO`s; the carpet `DECOFLAG`s are 3 less than the carpet `DECO`s; etc.